lib/core/src/Cardano/Wallet/Network.hs

```diff
@@ -63,8 +61,6 @@ instance Exception ErrNetworkUnreachable
 data ErrNetworkTip
     = ErrNetworkTipNetworkUnreachable ErrNetworkUnreachable
     | ErrNetworkTipNotFound
-    | ErrNetworkTipBlockNotFound (Hash "BlockHeader")
-    -- ^ The tip-block wasn't found. This would be surprising.
```

Comment:
Not needed. This is in the end, an implementation details which can be seen as `NetworkTipNotFound` from the external perspective.

Reply:
But they pretty much means the same for us in the end: there's no tip. We won't do anything special when the corresponding block isn't found bit treat that as if there was no tip.